scala-test-suite/src/test/resources/2.13.0-M4-pre-20d3c21/BlacklistedTests.txt

```diff
@@ -0,0 +1,143 @@
+## Do not compile
```

Comment:
I copied the 2.13.0-M3/BlacklistedTests file and added a few entries for new tests that don’t work in Scala.js.